integrators/explicit_embedded_runge_kutta_nyström_integrator.hpp

```diff
@@ -0,0 +1,109 @@
+﻿#pragma once
+
+#include <functional>
+#include <vector>
+
+#include "base/not_null.hpp"
+#include "integrators/motion_integrator.hpp"
+#include "quantities/named_quantities.hpp"
+
+namespace principia {
+
+using base::not_null;
+using quantities::Time;
+using quantities::Variation;
+
+namespace integrators {
+
+class ExplicitEmbeddedRungeKuttaNyströmIntegrator {
+ public:
+  ExplicitEmbeddedRungeKuttaNyströmIntegrator(
```

Comment:
Could we have a reference to some page (Witchipedia? Mathworld?) that describes the conventions for the parameters?